lua/damagelogs/shared/lang/brazilian_portuguese.lua

```diff
@@ -249,15 +248,13 @@ DamagelogLang.portugues = {
     AllPlayers = "Todos os jogadores",
     InvalidPlayerChat = "Jogador inválido!",
     CannotLeaveChat = "Você não pode sair como o único admin!",
-    Send = "Enviar",
```

Comment:
This is used in a few places
E.g. https://github.com/BadgerCode/tttdamagelogs/blob/master/lua/damagelogs/client/chat.lua#L561

Reply:
This is also in line 97.